assets/js/projects.json

```diff
@@ -50,31 +50,31 @@
   {
     "title": "Willo",
     "designer": "Jacob Witt",
-    "url": "../revival/willo",
+    "url": "../revival/willo/",
```

Comment:
I added this slash to make the filter I added to the JS work consistently. So whenever we add the remaining fonts, we want to make sure to include a closing slash in the `url` value of the JSON. 

For example, when we update Allie's JSON:
``` json
# Correct
"url": "../revival/allies-font/",

# Incorrect
"url": "../revival/allies-font",
```